packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -972,8 +987,20 @@ let ControlledKeyOpenCombobox = (props) => {
     }));
   };
 
+  // Handle removing selected key if input text is deleted
+  let onInputChange = (value: string) => {
+    setFieldState(prevState => ({
+      isOpen: prevState.isOpen,
+      selectedKey: value === '' ? null : prevState.selectedKey
+    }));
+  };
```

Comment:
Similar to how we require a user to provide `onSelectionChange` for a controlled input + open combobox, we now require a user to provide a `onInputChange` for a controlled key + open combobox